tests/test_pi_extension_dispatch.py

```
@@ -0,0 +1,383 @@
+    return subprocess.run(
+        [bun, "--eval", source],
+        cwd=root,
+        capture_output=True,
+        text=True,
+        check=False,
+        env=env,
+        timeout=20,
+    )
+
+
+def main() -> int:
```

Ruff (0.15.21)

[warning] Too many return statements (16 > 6)

(PLR0911)